core/tauri-macros/src/lib.rs

```diff
@@ -23,6 +23,12 @@ pub fn command(attributes: TokenStream, item: TokenStream) -> TokenStream {
   command::wrapper(attributes, item)
 }
 
+/// Accepts a list of commands functions. Creates a handler that allows commands to be called from JS with invoke().
```

Comment:
let's add an example here too :) want me to do it?

Reply:
i think something like `generate_handler![command_1, command_2, command_3]` would make sense

Reply:
Just added one :+1: